### PR TITLE
added separate inventories

### PR DESCRIPTION
**Pull request by 31i731**

### CODE REVIEW

File: source/actors/Player.hx [hunk 1]
```diff
@@ -9,7 +9,8 @@ import item.BaseItem;
 
 class Player extends FlxSprite {
 	public var speed:Float = 100;
-	public var inv:List<BaseItem>;
+	public var invActiveItems:List<BaseItem>;
```
Comment (cybercubes) [inline source/actors/Player.hx]: rename invActiveItems to activeItems

File: source/actors/Player.hx [hunk 2]
```diff
@@ -9,7 +9,8 @@ import item.BaseItem;
...
+	public var invPassiveItems:List<BaseItem>;
```
Comment (cybercubes) [inline source/actors/Player.hx]: rename invPassiveItems to passiveItems

File: source/actors/Player.hx [hunk 3]
```diff
@@ -91,27 +93,42 @@ class Player extends FlxSprite {
 	}
 
 	function activeItemUsing():Void {
```
Comment (cybercubes) [inline source/actors/Player.hx]: rename to useActiveItem

**Review by cybercubes — CHANGES_REQUESTED** [review]

**Review by 31i731 — COMMENTED** [review]
done

**Review by cybercubes — APPROVED** [review]

**Pull request merged**